src/wallet/wallet.cpp

```diff
@@ -833,9 +843,9 @@ bool CWallet::AddToWallet(const CWalletTx &wtxIn, bool fFromLoadWallet, CWalletD
             }
         }
 
-        //// debug print
-        LOGA("AddToWallet %s  %s%s\n", wtxIn.GetHash().ToString(), (fInsertedNew ? "new" : ""),
-            (fUpdated ? "update" : ""));
+        // Only useful if debugging wallet
+        // LOGA("AddToWallet %s  %s%s\n", wtxIn.GetHash().ToString(), (fInsertedNew ? "new" : ""),
```

Comment:
maybe we should add a new debug category for wallet related log statements

Reply:
yes, but I didn't want to do it just for this one log.